src/dropdown/dropdown.component.ts

```diff
@@ -52,7 +52,7 @@ import { hasScrollableParents } from "carbon-components-angular/utils";
 @Component({
 	selector: "ibm-dropdown",
 	template: `
-	<label *ngIf="label" [for]="id" class="bx--label">
+	<label *ngIf="label" [for]="id" class="bx--label" [ngClass]="{'bx--label--disabled': disabled}">
```

Comment:
```suggestion
	<label 
		*ngIf="label"
		[for]="id"
		class="bx--label"
		[ngClass]="{'bx--label--disabled': disabled}">
```

If there are more than 2 attributes make each attribute in a new line - this makes it easier to read & is the standard we have followed throughout the project.